src/StructuralSolvers/StructuralSolvers.jl

```diff
@@ -128,42 +127,35 @@ function _update!(ri_step::ResidualsIterationStep, ΔU_norm::Real, ΔU_rel::Real
 
     _step!(ri_step)
 
-    return ri_step
+    ri_step
 end
 
-"Updates the `ResidualsIterationStep` `i_step` given a `ConvergenceSettings` `cs`."
+"Updates the convergence criteria."
 function isconverged!(ri_step::ResidualsIterationStep, cs::ConvergenceSettings)
     ΔU_relᵏ, ΔU_nromᵏ = displacement_tol(ri_step)
```

Comment:
I suggest correcting all nroms by norms...

Reply:
Uh thanks for that!